README.md

```diff
@@ -46,6 +46,7 @@ This library currently supports the following cryptocurrencies and address forma
  - NEM (base32)
  - EOS
  - KSM (ss58)
+ - ADA (bech32)
```

Comment:
Can you provide reference link so that we can verify?

Reply:
https://forum.cardano.org/t/cip5-common-bech32-prefixes/35189
^^ @makoto